### DISCUSSION TITLE
Tapioca

### DISCUSSION THREAD

**Discussion by esdandreu** [charm-community/.github#7] [Show and tell]
Hi! I made a small library with some connectors I needed to add Bubbletea components to an existing CLI: https://github.com/esdandreu/glitter

![with glitter](https://github.com/esdandreu/glitter/raw/main/docs/log-progress.gif)

I found it difficult to add dynamic elements to existing CLI applications. It is particularly cumbersome if you already have a logging mechanism because logging can interfere with a dynamic element. Glitter solves this problem by becoming glue between a Bubbletea model and a logger's output.

![without glitter](https://github.com/esdandreu/glitter/raw/main/docs/log-progress_no-glitter.gif)

I think it fits into Charm's ecosystem and I was wondering if you were willing to adopt it. I am willing to transfer ownership to https://github.com/charm-community. It probably doesn't meet your quality standards yet, but the package is rather simple so it shouldn't be a problem to improve it to meet your quality expectations.

Let me know what you think,
Andreu

**Comment by meowgorithm** [comment]
Hey there! I personally think this would make a great addition. Are you open to changing the name? Glitter has some history at Charm and we've set the name aside for a possible future project.

**Reply by esdandreu** [reply]
Definitely! I chose glitter because it did match your naming style. I didn't know there was already a project with that name but it doesn't surprise me. Any suggestions?

This package is mostly related to bubbletea and I initially thought about it as an "anchored bubble" because it sits in the bottom of your terminal while your logging and prints float on top of it. Maybe something like "tapioca" because of bubbles sitting in the bottom of bubbletea?

**Reply by Delta456** [reply]
`tapioca` sounds like a great name.

**Reply by meowgorithm** [reply]
Yep, agreed — `tapioca` is good. So shall we transfer this in?

**Reply by esdandreu** [reply]
Ready when you are. I've just renamed the repository and the small documentation around it.

https://github.com/esdandreu/tapioca

If you agree I'll transfer the ownership to `charm-community`. I guess you'll need to accept it (I've never done it before). I would still like to be given some kind of administrative role to the repository so I can keep maintaining it.

Thank you!
Andreu

**Reply by Delta456** [reply]
@bashbunni has the authority to accept the transferring of the repo and also grant the administrative role.

**Reply by bashbunni** [reply]
@esdandreu Hey, happy to accept the repo whenever you'd like to transfer it. I can definitely give you the admin role on that repo

**Reply by esdandreu** [reply]
I requested the repository transfer to @bashbunni. I tried to transfer it directly to `charm-community` but it didn't work because I don't have repository creation access to it. Thank you all!

**Reply by bashbunni** [reply]
@esdandreu Awesome, I'll transfer it to the charm community on my end. Thanks so much!

**Reply by bashbunni** [reply]
@esdandreu Done! You have the admin role for this repository as well
